website/blog/2024-1-18-while-youre-busy-fixing-vulnerabilities/item.md

```diff
@@ -0,0 +1,115 @@
+
+| Security practice        | 2021 | 2023 |
+| ------------------------ | ---- | ---- |
+| Use of password policies | 79%  | 70%  |
+| Restricting admin rights | 75%  | 67%  |
+
+### At a minimum, resolve to review access this year
+
+Access management is number one—both in terms of things that get neglected, and a relatively simple thing to get on top of that can significantly reduce the impact of a breach.
+
+There are two nightmare access scenarios for a security engineer:
+
+-   **Everybody has admin status** ([Sysdig reports](https://sysdig.com/2023-cloud-native-security-and-usage-report/) that 90% of accounts have excessive permissions!)
+-   **Passwords are stored in plain text somewhere** (probably to enable shared logins)
+
+These practices are risky for multiple reasons. If everyone has admin permissions, suddenly a hacker only needs one compromised account to gain access to everything and do a lot of damage. And we all agree that storing passwords in a spreadsheet is not secure, and makes it hard to isolate who has access to what, and why.
+
+Big, mature corporations tend to have more strict permissions. One of the blessings/curses about working in startups is that they are usually more flexible—employees have fewer limitations on the scope of their role, which can be great for collaboration and people taking initiative. But that fluidity can make it harder to keep tabs on who should have access to what. Being overly permissive with admin status also opens the door to [shadow IT](https://en.wikipedia.org/wiki/Shadow_IT).
+
+![](./phish2.png)
+
+### Integrate access reviews into planning cycles
+
+If your company is making quarterly or even annual plans at the moment, it’s a great time to join in that process. The worst time to sync up with engineering/product management teams to understand their access needs is in the middle of a sprint.
+
+If you’re partnering with engineering up front though, it’s much easier to work together to understand their plans, so as the security function you can keep an eye out for unusual requests. Slotting access management reviews into an existing process helps to prevent the “security as blocker” problem (which often leads to friction between security and dev teams).
+
+### Set up an Identity and Access Management solution
+
+You can automate away some of the more routine/tedious aspects of Identity and Access Management (IAM) by implementing an identity provider. With SSO (Single Sign-on), you can map specific tasks to certain roles, and then further map refined-access roles to other roles, based on what tools a person is likely to need in their specific position at the company. For example, if someone has access to AWS, they automatically have access to your database. By using an SSO, especially one with [RBAC](https://goauthentik.io/docs/user-group-role/access-control) (Role-Based Access Control), you are only concerned with the initial setup being correct versus constant, ongoing reviews.
```

Comment:
"... you only need to be concerned with getting the initial setup correct rather than constant, ongoing monitoring and review."  ("Monitoring" is something that happens every day, review is a quarterly thing. Make the bad thing seems badder.  😊 )